app/models/user.rb

```diff
@@ -636,8 +636,12 @@ def bust_cache
     Users::BustCacheWorker.perform_async(id)
   end
 
+  # TODO: @citizen428 I don't want to completely remove this method yet, as we
+  # have similar methods in other models. But the previous implementation used
+  # three profile fields that we can't guarantee to exist across all Forems. So
+  # for now this method will just return an empty string.
```

Comment:
I think we can completely remove this method eventually and probably `search_score` as well. Unrelated to this PR though

Reply:
Yes, cleaning those up is on my todo list.